kairon/api/app/routers/bot.py

```diff
@@ -577,3 +578,31 @@ async def delete_http_action(action: str = Path(default=None, description="actio
         raise AppException(e)
     message = "HTTP action deleted"
     return Response(message=message)
+
+
+@router.post("/training-data", response_model=Response)
```

Comment:
rename this from training-data to training_examples/bulk

Reply:
/data/bulk